apps/src/storage/firebaseStorage.js

```diff
@@ -141,7 +138,7 @@ function fixTableName(tableName, onError) {
       type: WarningType.TABLE_RENAMED,
       msg:
         `The table was renamed from "${tableName}" to "${newTableName}" because the characters ` +
-        '".", "$", "#", "[", "]", and "/" are not allowed in table names.'
+        '"$", "#", "[", "]", and "/" are not allowed in table names.'
```

Comment:
NIT: Should this be added to i18n?

Reply:
None of the error messages for data features are translated, so I'll leave that as a future task for whenever we translate App Lab.